src/pe/debug.rs

```diff
@@ -0,0 +1,120 @@
+use scroll::{self, Pread};
+use error;
+
+use pe::section_table;
+use pe::utils;
+use pe::data_directories;
+
+#[derive(Debug, PartialEq, Copy, Clone, Default)]
+pub struct DebugData<'a> {
+    pub image_debug_directory: ImageDebugDirectory,
+    pub codeview_pdb70_debug_info: Option<CodeviewPDB70DebugInfo<'a>>,
+}
+
+impl<'a> DebugData<'a> {
+    pub fn parse(bytes: &'a [u8], dd: &data_directories::DataDirectory, sections: &[section_table::SectionTable]) -> error::Result<Self> {
+        let image_debug_directory = ImageDebugDirectory::parse(bytes, dd, sections)?;
+        let codeview_pdb70_debug_info = CodeviewPDB70DebugInfo::parse(bytes, &image_debug_directory)?;
+
+        Ok(DebugData{
+            image_debug_directory,
```

Comment:
Looks like that's 1.17: https://github.com/rust-lang/rust/commit/1b6f1fe9dd29c97fca66a7377f367d0beb327b58

I'll fix.